en/ajax.texy

```diff
@@ -211,3 +211,28 @@ In this case, the simplest way is to wrap the whole list in one more snippet and
 \--
 
 The same goes for deleting an item. It would be possible to send empty snippet, but usually lists can be paginated and it would be complicated to implement deleting one item and loading another (which used to be on a different page of the paginated list).
+
+
+Sending parameters to component
+===============================
+
+If we sending parameters to components by AJAX request, whether signal parameters or persistent parameters, we must set global name of parameters for request, witch contains name of component. Full name of parameter return method `getParameterId()`.
```

Comment:
Nejsem úplně angličtinář, ale nemělo by to být spíš nějak takto?

When we send parameters to a component via AJAX request, whether signal parameters or persistent parameters, we must provide their global name, which also contains the name of the component. The full name of parameter returns the `getParameterId()` method.

Reply:
jj, díky za opravu 👍